epoxy-sample/src/main/java/com/airbnb/epoxy/sample/models/ColorModel.java

```diff
@@ -54,6 +61,31 @@ public void bindChanges(@NonNull ColorHolder holder,
     }
   }
 
+  @Nullable
+  @Override
+  public EpoxyItemAnimation bindChangesAnimated(@NonNull ColorHolder view,
+                                                @NonNull EpoxyModel<?> previouslyBoundModel,
+                                                @Nullable EpoxyItemAnimation interruptedAnimation) {
+
+    int currentColor = ((ColorModel) previouslyBoundModel).color;
+    float currentRotation = 0f;
+
+    if(interruptedAnimation != null) {
+      AnimatorSet previousAnim = (AnimatorSet) ((EpoxyItemAnimation.AnimatorWrapper) interruptedAnimation).wrappedAnimator;
```

Comment:
I mean you could technically get the previous state from the viewHolder too, so `previousModel` is just as redundant as `interruptedAnimation`, apart from being a little more convenient.

 `DefaultItemAnimator`, and probably most third-party animators who are inspired by it, use `getChangeDuration` to play change animations after removals and before additions. Looks somewhat better but it's not necessary. 
Animatons are also not run immediately when `animate*` is called but in a batch in `runPendingAnimations` when RV tells it to. Not sure if this is really needed. It seems to work mostly fine when you run them in `bindViewHolder` and stop in `unbind`.